libraries-data-2/pom.xml

```diff
@@ -216,5 +216,5 @@
             </plugin>
         </plugins>
     </build>
-
+    
```

Comment:
Don't need the whitespace change. And this is from a completely different module.

Reply:
Removed extra line. For a brief moment, bookkeper was there..